src/frontend/komponenter/Behandling/Inngangsvilkår/Inngangsvilkår.tsx

```diff
@@ -61,11 +61,8 @@ const Inngangsvilkår: FC<Props> = ({ behandlingId }) => {
     }, [postInngangsvilkårSuksess]);
 
     const ferdigVurdert = (behandlingId: string): any => {
-        const postInngangsvilkår = (): Promise<Ressurs<string>> => {
-            return axiosRequest<any, any>({
-                method: 'POST',
-                url: `/familie-ef-sak/api/vurdering/${behandlingId}/vilkar/fullfor`,
-            });
+        const postInngangsvilkår = () => {
```

Comment:
Notér i refaktorer-boka; Slett all kode som handler om postInngangsvilkår / feilmelding / ferdigVurdert

Hvis vi skal ha knapp så skal den bare gjøre history.push - men tror vi kan fjerne knappen også :)

Reply:
Notert :)